### PR TITLE
Update documentation for 1.21 actions

### LINKED ISSUE #26309: "Variable" set for gitea action not accessible in job
### Description

When setting a custom variable for gitea action in "repo -> settings -> actions -> variables -> "Variables Management"" it is not accessible in the job. 

I have tried ${{VARNAME}}, ${{github.VARNAME}}, ${{env.VARNAME}}, and ${{gitea.VARNAME}}.

act_runner is on a different server.

### Gitea Version

1.21.0+dev-437-gb9baed2c74

### Can you reproduce the bug on the Gitea demo site?

Yes

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

archlinux

### How are you running Gitea?

installed gitea-git package from archlinux AUR

### Database

SQLite

### PR DESCRIPTION
**Pull request by sillyguodong**
As title.
Close #26309
Related to #24724, #24806

### CODE REVIEW

**Review by wolfogre — APPROVED** [review]

**Review by lunny — APPROVED** [review]

File: docs/content/usage/actions/act-runner.en-us.md [hunk 1]
```diff
@@ -245,8 +245,7 @@ You can find more useful images on [act images](https://github.com/nektos/act/bl
 If you want to run jobs in the host directly, you can change it to `ubuntu-22.04:host` or just `ubuntu-22.04`, the `:host` is optional.
 However, we suggest you to use a special name like `linux_amd64:host` or `windows:host` to avoid misusing it.
 
-One more thing is that it is recommended to register the runner if you want to change the labels.
-It may be annoying to do this, so we may provide a better way to do it in the future.
+After Gitea 1.21 released, you can change labels by modfiying `container.labels` in configuration file (if you don't have a configuration file, please refer to [configuration tutorials](#configuration)), and runner will declare the new labels which you defined in configuration file after executing `./act_runner daemon --config config.yaml`.
```
Comment (delvh) [inline docs/content/usage/actions/act-runner.en-us.md:248]:
```suggestion
Starting with Gitea 1.21, you can change labels by modifying `container.labels` in the runner configuration file (if you don't have a configuration file, please refer to [configuration tutorials](#configuration)).
The runner will use these new labels as soon as you restart it, i.e., by calling `./act_runner daemon --config config.yaml`.
```

File: docs/content/usage/actions/act-runner.en-us.md [hunk 2]
```diff
@@ -261,3 +260,32 @@ After you have registered the runner, you can run it by running the following co
 The runner will fetch jobs from the Gitea instance and run them automatically.
 
 Since act runner is still in development, it is recommended to check the latest version and upgrade it regularly.
+
+## Configuration variable
+
+You can create configuration varibales with user, organization, repository level. And the level of the variable depends on which setting panel you created in.
```
Comment (delvh) [inline docs/content/usage/actions/act-runner.en-us.md:266]:
```suggestion
You can create configuration variables on the user, organization, and repository level.
The level of the variable depends on where you created it.
```

File: docs/content/usage/actions/act-runner.en-us.md [hunk 3]
```diff
@@ -261,3 +260,32 @@ After you have registered the runner, you can run it by running the following co
...
+
+- Varibale names must not start with the `GITHUB_` and `GITEA_` prefix.
+
+- Varibale names must not start with a number.
+
+- Varibale names are not case-sensitive.
+
+- Varibale names must be unique at the level they are created at.
+
+- Varibale names must not be 'CI'.
```
Comment (delvh) [inline docs/content/usage/actions/act-runner.en-us.md:282]:
```suggestion
- Variable names can only contain alphanumeric characters (`[a-z]`, `[A-Z]`, `[0-9]`) or underscores (`_`). Spaces are not allowed.

- Variable names must not start with the `GITHUB_` and `GITEA_` prefix.

- Variable names must not start with a number.

- Variable names are case-insensitive.

- Variable names must be unique at the level they are created at.

- Variable names must not be `CI`.
```

File: docs/content/usage/actions/act-runner.en-us.md [hunk 4]
```diff
@@ -261,3 +260,32 @@ After you have registered the runner, you can run it by running the following co
...
+
+### Using varibales
+
+After creating configuration varibales, they will be automatically filled in the `vars` context. They are available to you with expression like `{{ vars.VARIABLE_NAME }}` in workflow.
```
Comment (delvh) [inline docs/content/usage/actions/act-runner.en-us.md:286]:
```suggestion
After creating configuration variables, they will be automatically filled in the `vars` context. They can be accessed through expressions like `{{ vars.VARIABLE_NAME }}` in the workflow.
```

File: docs/content/usage/actions/act-runner.en-us.md [hunk 5]
```diff
@@ -261,3 +260,32 @@ After you have registered the runner, you can run it by running the following co
...
+
+### Precedence
+
+If a variable with the same name exists at multiple levels, the variable at the lowest level takes precedence(the level of organization and user is higher than repository's).
+For example, if an organization-level variable has the same name as a repository-level variable, then the repository-level variable takes precedence.
```
Comment (delvh) [inline docs/content/usage/actions/act-runner.en-us.md:291]:
```suggestion
If a variable with the same name exists at multiple levels, the variable at the lowest level takes precedence:
A repository variable will always be chosen over an organization/user variable.
```

**Review by delvh — CHANGES_REQUESTED** [review]

**Comment by delvh** [comment]
Oh.

**Comment by sillyguodong** [comment]
I will create a new PR to update😂

**Pull request merged**